openhtf/__init__.py

```diff
@@ -45,6 +46,106 @@ class InvalidTestPhaseError(Exception):
   """Raised when an invalid method is decorated."""
 
 
+def Test(*phases, **metadata):
+  return TestInfo(phases, metadata=metadata)
+
+
+class TestInfo(mutablerecords.Record(
```

Comment:
While I agree with #106 in general, I don't agree here. This is a TestInfo class, the Execute method is a convenience wrapper around two other classes that do actual work. I would prefer an `openhtf.ExecuteTest(...)` function to do that if I wanted to be a purist